backend/tests.py

```diff
@@ -236,6 +236,15 @@ def get_workshop(self, id):
                           content_type="application/json")
         return json.loads(rv.get_data(as_text=True))
 
+    def get_latest_workshops(self):
```

Comment:
This is a good sanity check test. Consider adding some additional tests that match the requirements.  Should we only ever get six recommended workshops back?  Should those workshops be the latest workshops available? etc...